R/tokens.R

```diff
@@ -157,13 +158,10 @@ tokens <-  function(x,
                     include_docvars = TRUE,
                     padding = FALSE,
                     verbose = quanteda_options("verbose"),
-                    ...) {
+                    ...,
+                    xptr = FALSE) {
```

Comment:
I like this, it's simpler, and suggests we could flip it to `TRUE` eventually.
I know you dislike the idea of testing this in a 3.x version, but if we sneaked it in this way, we could allow tests and comparisons with just one toggle. Maybe 4.0 would set it to `TRUE`?

Reply:
I have to see how many people will really use the `tokens_xptr` object. If a corpus contains only a few thousands documents, it is easier to use the classic object.